integration-tests/src/test/kotlin/org/stellar/anchor/platform/AuthIntegrationTest.kt

```diff
@@ -0,0 +1,195 @@
+package org.stellar.anchor.platform
+
+import java.util.concurrent.TimeUnit
+import kotlin.test.assertEquals
+import okhttp3.OkHttpClient
+import okhttp3.Request
+import okhttp3.RequestBody
+import org.junit.jupiter.api.AfterAll
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.BeforeAll
+import org.junit.jupiter.api.assertThrows
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.CsvSource
+import org.stellar.anchor.api.exception.SepNotFoundException
+import org.stellar.anchor.apiclient.PlatformApiClient
+import org.stellar.anchor.auth.AuthHelper
+import org.stellar.anchor.auth.JwtService
+import org.stellar.anchor.platform.AbstractIntegrationTest.Companion.ANCHOR_TO_PLATFORM_SECRET
+import org.stellar.anchor.platform.AbstractIntegrationTest.Companion.PLATFORM_TO_ANCHOR_SECRET
+import org.stellar.anchor.util.GsonUtils
+import org.stellar.anchor.util.OkHttpUtil
+
+open class AbstractAuthIntegrationTest {
+  companion object {
+    private val jwtService =
+      JwtService(null, null, null, PLATFORM_TO_ANCHOR_SECRET, ANCHOR_TO_PLATFORM_SECRET)
+    internal val jwtAuthHelper = AuthHelper.forJwtToken(jwtService, 10000)
+    internal val apiAuthHelper = AuthHelper.forApiKey(ANCHOR_TO_PLATFORM_SECRET)
+    internal val nonAuthHelper = AuthHelper.forNone()
+    internal val nonAuthPlatformClient = PlatformApiClient(nonAuthHelper, "http://localhost:8085")
+    internal lateinit var testProfileRunner: TestProfileExecutor
+  }
+
+  init {}
+}
+
+internal class JwtAuthIntegrationTest : AbstractAuthIntegrationTest() {
+  companion object {
+    @BeforeAll
+    @JvmStatic
+    fun setup() {
+      testProfileRunner =
+        TestProfileExecutor(
+          TestConfig(profileName = "default").also {
+            // enable platform server jwt auth
+            it.env["platform_server.auth.type"] = "jwt"
+          }
+        )
+      testProfileRunner.start()
+    }
+
+    @AfterAll
+    @JvmStatic
+    fun breakdown() {
+      testProfileRunner.shutdown()
+    }
+  }
+
+  private val jwtPlatformClient: PlatformApiClient =
+    PlatformApiClient(jwtAuthHelper, "http://localhost:8085")
+
+  // TODO - to be deprecated by platformAPI client
+  private val httpClient: OkHttpClient =
+    OkHttpClient.Builder()
+      .connectTimeout(10, TimeUnit.MINUTES)
+      .readTimeout(10, TimeUnit.MINUTES)
+      .writeTimeout(10, TimeUnit.MINUTES)
+      .build()
+
+  @ParameterizedTest
+  @CsvSource(
+    value =
+      [
```

Comment:
Can you make this list as a constant somewhere?

Reply:
Changed.

Reply:
FYI: you can make a constant a whole array :)